src/components/tiles/text/text-tile.tsx

```diff
@@ -186,40 +200,44 @@ export default class TextToolComponent extends BaseComponent<ITileProps, IState>
     // used here for a while now and cypress tests depend on it. Should transition
     // to using 'text-tool-editor' for these purposes moving forward.
     const classes = `text-tool text-tool-editor ${editableClass}`;
-
     if (!editorValue) return null;
 
     return (
       // Ideally, this would just be 'text-tool' for consistency with other tools,
       // but 'text-tool` is used for the internal editor (cf. 'classes' above),
       // which is used for cypress tests and other purposes.
-      <div className={`text-tool-wrapper ${readOnly ? "" : "editable"}`}
-        data-testid="text-tool-wrapper"
-        ref={elt => this.textTileDiv = elt}
-        onMouseDown={this.handleMouseDownInWrapper}>
-        <TextToolbarComponent
-          documentContent={documentContent}
-          tileElt={tileElt}
-          scale={scale}
-          selectedButtons={selectedButtons || []}
-          editor={this.editor}
-          onIsEnabled={this.handleIsEnabled}
-          onRegisterTileApi={this.handleRegisterToolApi}
-          onUnregisterTileApi={this.handleUnregisterToolApi}
-        />
-        <SlateEditor
-          className={classes}
-          onEditorRef={this.handleEditorRef}
-          value={editorValue}
-          placeholder={placeholderText}
-          readOnly={readOnly}
-          plugins={this.plugins}
-          onValueChange={this.handleChange}
-          onFocus={this.handleFocus}
-          onBlur={this.handleBlur}
-
-        />
-      </div>
+      // FIXME: replace this provider with one at the tile level so we get it for free.
```

Comment:
We had chatted about making this a generic provider on Tiles to provide the content but I ran out of time so Ive left it as a FIXME.